comtypes/server/register.py

```diff
@@ -310,21 +310,16 @@ def _registry_entries(self, cls):
         ):
             exe = sys.executable
             if " " in exe:
-                exe = '"%s"' % exe
+                exe = f'"{exe}"'
             if not hasattr(sys, "frozen"):
                 if not __debug__:
-                    exe = "%s -O" % exe
+                    exe = f"{exe} -O"
                 script = os.path.abspath(sys.modules[cls.__module__].__file__)
                 if " " in script:
-                    script = '"%s"' % script
-                append(
-                    HKCR,
-                    "CLSID\\%s\\LocalServer32" % reg_clsid,
-                    "",
-                    "%s %s" % (exe, script),
-                )
+                    script = f'"{script}"'
+                append(HKCR, rf"CLSID\{reg_clsid}\LocalServer32", "", f"{exe}")
```

Comment:
Originally, it was as below, so `f"{exe}"` needs to be changed to `f"{exe} {script}"`.

https://github.com/enthought/comtypes/blob/1ea74f5a68c4f1e0ee511ba3591e2204688ec071/comtypes/server/register.py#L318-L327
```diff
-               append(HKCR, rf"CLSID\{reg_clsid}\LocalServer32", "", f"{exe}")
+               append(HKCR, rf"CLSID\{reg_clsid}\LocalServer32", "", f"{exe} {script}")
```